libtransmission/rpc-server.cc

```diff
@@ -658,6 +701,18 @@ void start_server(tr_rpc_server* server)
     auto const address = server->get_bind_address();
     auto const port = server->port();
 
+#ifdef WITH_OPENSSL
+    SSL_CTX* m_ctx = nullptr;
```

Comment:
Nit: what does the `m_` prefix signify in two places? It might indicate member variable naming convention but a) it's not a member variable, and b) our convention for member variables is `_` suffix instead.

Reply:
change to `ctx`